crates/streamer/src/streamer/tag_writers.rs

```diff
@@ -344,13 +344,9 @@ impl TryFrom<(Headers<'_>, &StreamingConfig)> for EncodeWriterConfig {
 
         let config = EncodeWriterConfig {
             threads: config.encoder_threads,
-            width: width
-                .map(u32::try_from)
-                .ok_or_else(|| anyhow::anyhow!("No width specified"))??,
-            height: height
-                .map(u32::try_from)
-                .ok_or_else(|| anyhow::anyhow!("No height specified"))??,
-            codec: codec.ok_or_else(|| anyhow::anyhow!("No codec specified"))?,
+            width: width.map(u32::try_from).context("No width specified")??,
+            height: height.map(u32::try_from).context("No height specified")??,
+            codec: codec.context("No codec specified")?,
```

Comment:
https://github.com/Devolutions/IronRDP/blob/eb499de352d7da03f17dcd10f61c764c0bdeb81d/STYLE.md#formatting-of-error-messages

Reply:
updated